src/store/modules/dashboard/type.ts

```diff
@@ -7,6 +7,13 @@ export const VIEWERS_TYPE = Object.freeze({
 
 export type ViewersType = typeof VIEWERS_TYPE[keyof typeof VIEWERS_TYPE];
 
+export const SCOPE_TYPE = Object.freeze({
+    DOMAIN: 'DOMAIN',
+    PROJECT: 'PROJECT',
+});
```

Comment:
suggestion: please use `as const` rather than `Object.freeze` (just for convention)

```suggestion
export const SCOPE_TYPE = {
    DOMAIN: 'DOMAIN',
    PROJECT: 'PROJECT',
} as const;
```